debugger/src/tpage.c

```diff
@@ -196,27 +223,45 @@ void tpage_pack_widgets(gboolean tabbed)
 	{
 		GtkWidget *lbox, *rbox, *hbox;
 
+#if GTK_CHECK_VERSION(3, 0, 0)
+		root = gtk_box_new(GTK_ORIENTATION_HORIZONTAL, SPACING);
+		gtk_box_set_homogeneous(GTK_BOX(root), TRUE);
```

Comment:
Here also the line could be used for both versions maybe.

Reply:
Same as above : it would be redundant in the GTK2 code.

Reply:
Ok, let's keep it to.